cmd/otelcol/Dockerfile.windows

```
@@ -27,6 +27,23 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${env:JMX_METRIC_GATHERER_RELEASE}/opentelemetry-jmx-metrics.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JDK_VERSION="11.0.19_7"
+ARG OPENJDK_BASE_URL="https://github.com/adoptium/temurin11-binaries/releases/download"
+ENV JAVA_HOME="C:\java\jdk-11.0.19+7"
```

```suggestion
ENV JAVA_HOME="C:\java\jdk-11.0.20+8"
```
whenever `JDK_VERSION` is updated.

May also be useful to dynamically set this path based on `JDK_VERSION`, or just rename the extracted jdk directory to something like `C:\java\jdk`.

---

Updated, and I'll try to re-use the version since the path is a sub-directory of the downloaded JDK, not the top level path.